libs/components/core/src/index.ts

```diff
@@ -17,6 +17,7 @@ export { SkyAffixer } from './lib/modules/affix/affixer';
 
 export { SkyContentInfoProvider } from './lib/modules/content-info-provider/content-info-provider';
 export { SkyContentInfo } from './lib/modules/content-info-provider/content-info';
+export { SkyContentInfoDescriptor as SkyDescriptor } from './lib/modules/content-info-provider/descriptor';
```

Comment:
The `as SkyDescriptor` isn't needed here. It's probably an artifact of renaming the class.